.env_SAMPLE

```diff
@@ -7,9 +7,6 @@
 # Django Application Server.
 ############################
 
-export DJANGO_ENV=base
-#export DJANGO_STATIC_ROOT=<path_to_static_files>
-
```

Comment:
still going to need `DJANGO_STATIC_ROOT` for setting the static_url on flapjack/refresh servers